contact.html

```diff
@@ -3,3 +3,4 @@ <h1>Super Contact Modified...</h1>
 
 Author: Marcus Rangel.
 
+License: MIT.
```

Comment:
The license type should be in bold style, could you correct it???

Reply:
ok!!